packages/postcss-ordered-values/src/__tests__/index.js

```diff
@@ -581,3 +581,19 @@ test(
     'border-inline: 5px dashed blue; border-inline-end: .5em solid red; border-block-end: 5px dashed blue; border-block: 5px solid red;border-block-start: 1px solid rgba(0, 30, 105, 0.8);'
   )
 );
+
+test(
+  'should order columns only if unit is mentioned',
+  processCSS(
+    'h1 {columns: 2 auto;columns: auto 12em;columns: auto auto;columns: auto 12em auto 13em;}',
```

Comment:
Is `auto 12em auto 13em` a valid value? I'd be wary of adding tests around invalid syntax.

Reply:
no it's not, sorry !!!
I will change it